exporter.go

```diff
@@ -185,6 +205,9 @@ For NGINX, the stub_status page must be available through the URI. For NGINX Plu
 	nginxRetries = flag.Uint("nginx.retries",
 		defaultNginxRetries,
 		"A number of retries the exporter will make on start to connect to the NGINX stub_status page/NGINX Plus API before exiting with an error. The default value can be overwritten by NGINX_RETRIES environment variable.")
+	constLabels = flag.String("prometheus.const-labels",
```

Comment:
Could we have a flag similar to `createPositiveDurationFlag` or `createPositiveDurationFlag`? this way will get consistent validation and error reporting.